.github/workflows/tests.yml

```diff
@@ -4,20 +4,20 @@ name: Tests
 # Controls when the action will run. Triggers the workflow on push or pull request
 # events but only for the wanted branches
 on:
-  push:
-    branches:
-      - master
-      - main
   pull_request:
-    branches:
-      - master
-      - main
+  push:
+    branches: [master, main]
```

Comment:
Note that this will no longer run CI on pull requests from forks. Since this is a public repo I don't think this new behavior is desired

Reply:
This syntax is actually identical. It is just a shrorter way of writing the same.

Reply:
Ah you're right, my bad. I was reviewing on mobile and missed that `pull_request:` wasn't removed.